packages/ses/src/whitelist.js

```diff
@@ -932,14 +935,22 @@ export const whitelist = {
       values: 'boolean',
       // Failed tc39 proposal
       // Seen on FF Nightly 88.0a1
-      at: false,
+      at: 'boolean',
```

Comment:
Saw the false, and even though `at` is a withdrawn experiment, I wonder if we should always err on unscopables towards inclusion?

Reply:
If the property is present on Array.prototype, we certainly want it to be unscopable. But, lockdown will remove it if it is present.

Reply:
I just checked, and we do list `Array.prototype.at` as not to be removed. Given your remarks, it seems that not removing it on unscopables was indeed a correct fix, so Resolving. However, I'm curious: Given

> If the property is present on Array.prototype, we certainly want it to be unscopable.

Shouldn't all methods on Array.prototype already be unscopable in plain JS? Node JS does not even come close:

```
> gopn = Object.getOwnPropertyNames
[Function: getOwnPropertyNames]
> gopn(Array.prototype).length
34
> gopn(Array.prototype[Symbol.unscopables]).length
11
```

Reply:
My understanding is that methods of arrays that predates the introduction of unscopables must not be unscopable, and all that followed must be unscopable, such that all legacy code that uses arrays as with block contexts continue to work, both in the case where they intentionally invoke a method of the array and all the cases where a new array method would overshadow a lexical function. 

with (arr) push(10)

function flatMap() {}
with (arr) flatMap()

Reply:
Makes sense, thanks!